cypress/support/commands/item.js

```diff
@@ -124,7 +127,7 @@ Cypress.Commands.add(
     cy.fillBaseItemModal({ name }, { confirm: false });
 
     cy.get(`#${ITEM_FORM_APP_URL_ID}`).click();
-
+    // todo: can not edit name field in modal
```

Comment:
We could add the textfield for the name in the edit modal if you want. Is it referring to that?

Reply:
I put that there to remind me of adding a test that would try to change the app name. I obviously forgot to add it. Doing so in the next commit.